src/Features/Navigation/CoreMenu.php

```diff
@@ -140,45 +134,62 @@ public static function get_items() {
 				continue;
 			}
 
-			$wca_items[] = array_merge(
+			$path = isset( $page['path'] ) ? $page['path'] : null;
+			$item = array_merge(
 				array(
-					'id'         => 'wc_admin-' . $page['path'],
-					'url'        => $page['path'],
+					'id'         => 'wc_admin-' . $path,
+					'url'        => $path,
 					'title'      => $page['title'][0],
-					'capability' => $page['capability'],
+					'capability' => isset( $page['capability'] ) ? $page['capability'] : 'manage_woocommerce',
 				),
 				$page['nav_args']
 			);
+
+			// Don't allow top-level items to be added to the primary menu.
+			if ( ! isset( $item['parent'] ) || 'woocommerce' === $item['parent'] ) {
+				$item['menuId'] = 'plugins';
```

Comment:
If we want to enforce this by switching the `menuId` automatically, maybe we should still print some kind of warning in the error log?

Reply:
@joelclimbsthings I think this could result in a pretty long error log, even when the expected occurs (wanting to register in the plugins menu).

Or do you think instead we should require `menuId` to be `plugins` if a parent does not exist and skip adding the item if that's the case?

Reply:
Going to get this merged so we can unblock other PRs.  Let me know what you think on the above @joelclimbsthings and we can follow-up on this.

Reply:
That's true, I think the condition would have to be a bit different with a warning. My thought was mainly if the `menuId` was actually set to something other than plugins, and we switched it, a warning would be more justified. 
Something like this, within that condition:
```
if ( ! isset( $item['parent'] ) || 'woocommerce' === $item['parent'] ) {
    if( $item['menuId'] && 'plugins' !== $item['menuId'] ) {
        //print warning with error_log()
    }
    $item['menuId'] = 'plugins';
}
```

Or do you think that would still be too noisy @joshuatf ?